src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_DateAndTime.vb

```diff
@@ -89,5 +91,35 @@ class c
                 Await state.AssertNoCompletionSession()
             End Using
         End Function
+
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestExample1(showCompletionInArgumentLists As Boolean) As Task
```

Comment:
Curious, are these command handler tests because they're sensitive to the location of the caret? We can't test those as direct completion provider tests?

Reply:
i hate the direct completion provider tests.  because tey effectively are presuming things about how those providers will be used.  i really want to treat it as a black and say: this is the experience the customer gets.  i would prefer 100% of all completoin tests move to this form.